docs/tutorial/overview.md

```diff
@@ -21,15 +21,15 @@ As you go through the tutorial pages, you will encounter embedded boxes with dif
 
 NOMAD is available in two flavours to address the diverse needs of scientists: **Central NOMAD** and **NOMAD Oasis**. To learn more about these solutions, refer to our [web page](https://nomad-lab.eu/){:target="_blank"}.
 
-Our tutorial documentation covers basic usage that can be performed in both Central NOMAD and NOMAD Oasis. These tasks range from basic operations using the graphical user interface (GUI) to a certain level of customization (level 1) using YAML files, which do not require local installations.
+Our tutorial documentation covers basic usage that can be performed in both Central NOMAD and NOMAD Oasis. These tasks range from basic operations using the graphical user interface (GUI) to a certain level of tailorling electronic lab notebooks using YAML files, which do not require local installations.
 
-Additionally, the tutorials include in-depth customization (level 2) of NOMAD Oasis using NOMAD Plugins. However, more advanced topics such as installing an Oasis, setting up an internal Keycloak instance, managing a local file system, or adding tools to NORTH are not currently covered in the tutorials. Users interested in these topics should refer to other sections of the documentation, such as:
+Additionally, the tutorials include in-depth customization of NOMAD Oasis using NOMAD Plugins. However, more advanced topics such as installing an Oasis, setting up an internal Keycloak instance, managing a local file system, or adding tools to NORTH are not currently covered in the tutorials. Users interested in these topics should refer to other sections of the documentation, such as:
 
 - [How-to guides](../howto/overview.md) for step-by-step instructions,
```

Comment:
I don't really think these are necessary, they are clearly visible in the side navigation bar and also on the docs landing page